layouts/shortcodes/feature-gate-list.html

```diff
@@ -0,0 +1,61 @@
+<table>
+  <thead>
+   <tr>
+     <th>Feature name</th>
+     <th>Default value</th>
+     <th>Stage</th>
+     <th>Since</th>
+     <th>Until</th>
+     <th>Debug</th>
+   </tr>
+  </thead>
+  <tbody>
+{{- range $featureGate := $.Site.Data.features.featureList.featureGateList -}}
+  {{- range $versionUnderConsideration := (seq 1 21 ) -}}
+    {{ if eq (index (split $featureGate.alphaFromVersion "." ) 1) (printf "%d" $versionUnderConsideration )}}
+  <tr class="feature-stage-row" data-featurestage="alpha">
```

Comment:
`data-featurestage` is to allow a future change where you can hide rows using CSS

For example, you might only care about graduated features. Or you might only want to know what changed in the release you're reading the docs for.

Feature gates that are disabled because they were deprecated then removed might be hidden by default. That kind of idea.

Reply:
indeed, this would make it easier to sort through the feature gates. If a feature gate has been disabled for that version it could probably be hidden by default since it's no longer relevant.